### ISSUE TITLE
top_cpu not working if language not english

### ISSUE THREAD

**Issue by dflvunoooooo** [roger-/hass-sysmon#10]
The top_cpu sensor does not work if the language is not set to english. The `command_ind` returns empty because `COMMAND` is not found. In my case it is called `BEFEHL`.

**Comment by roger-** [comment]
Hmm not sure how to handle this. IIRC different systems are not consistent in the ordering of the top output hence the need to find the column index.

I’d rather not have to have special cases for each language. 

Think you might need to handle this with your custom translation until we can figure out a better approach.

**Comment by dflvunoooooo** [comment]
Yeah, the only way I can think of is a gigantic loop for every language, where it is named differently.
